packages/typescript/internal/ts_project.bzl

```diff
@@ -421,6 +443,35 @@ def ts_project_macro(
         )
         extra_deps.append("_validate_%s_options" % name)
 
+    is_windows = select({
+        "@bazel_tools//src/conditions:host_windows": True,
+        "//conditions:default": False,
+    })
+    supports_workers = supports_workers and not is_windows
```

Comment:
oops, this is checking if the `select` is false, not the result of it - so workers are totally turned off in this PR as-is. I'll push a fix...